assets/js/modules/analytics/components/setup/SetupForm.js

```diff
@@ -82,7 +84,10 @@ export default function SetupForm( { finishSetup } ) {
 	return (
 		<form className="googlesitekit-analytics-setup__form" onSubmit={ submitForm }>
 			<StoreErrorNotice moduleSlug="analytics" storeName={ STORE_NAME } />
+			<ExistingGTMPropertyError />
```

Comment:
Same here, let's place this in `SetupMain` instead. If this error is shown, nothing else should be shown entirely during setup, the dropdowns and button to proceed shouldn't be there, similar to how it is for an `ExistingTagError`.

Reply:
Updated.